jetbrains-core/tst/software/aws/toolkits/jetbrains/ui/tree/AsyncTreeModelTest.java

```diff
@@ -62,7 +61,7 @@
   public ProjectRule projectRule = new ProjectRule();
 
   @Test
-  public void testAggressiveUpdating() {
+  public void testAggressiveUpdating() throws TimeoutException {
```

Comment:
Why is this a java file...

Reply:
That would be my doing. These are taking from IJ for our fork of async tree model and the like. We can remove when base is 2020.1